components/contextmenu/events.md

```diff
@@ -10,9 +10,93 @@ position: 30
 
 # Events
 
-This article explains the events available in the Telerik Context Menu for Blazor:
+This article describes the events generated by the Telerik Context Menu for Blazor:
 
-* [OnClick](#onclick)
+* [`OnClick`](#onclick)
+* [`OnItemRender`](#onitemrender)
```

Comment:
The events are sorted alphabetically, which is OK, but then the sections are not.